lib/qmp.ml

```diff
@@ -29,11 +29,20 @@ type enabled = {
   present: bool;
 }
 
+type vnc = {
+  enabled : bool;
+  auth    : string;
+  family  : string;
+  service : string;
```

Comment:
Given that we are converting service to `int` on the destination, I'd rather have it here. (We do it for timestamp already)